test/strong_migrations_test.rb

```diff
@@ -368,6 +374,10 @@ def test_down
     assert_safe SafeUp, direction: :down
   end
 
+  def test_add_foreign_key
+    assert_safe AddForeignKey
```

Comment:
Temporarily flipped this to `assert_safe` so I can inspect the "suggested change" messaging. Should flip this back before calling the work here done.